kshebekoMv/task2/XPathExercises1.csv

```diff
@@ -0,0 +1,38 @@
+Xpath Tasks,Authors.xml,Authors2.xml (with namespaces);;;;;;;;;;;
+Normal:,,;;;;;;;;;;;
+all elements,,;//*;;;;;;;;;;
+"all ""author"" elements,,";//author//*;;;;;;;;;;
+"the root element ""authors"" - 2 variants,,"; /authors and //author/ancestor::authors and /*;;;;;;;;;;
+the same as 3,  but this time without using the name of the element,,;//author/..;;;;;;;//author/parent::authors;;;?
+all elements enclosed by elements  /authors/author,,;//author/*;;;;;;;;;;
+"all ""name"" elements (without using names of parent elements) ,,";//name;;;;;;;;;;
+all elements which have 1 ancestors,,;//ancestor-or-self::* and //ancestor::*;;;;;;;//author[*];;;
+"all ""period"" attributes of ""author"" elements - 2 variants,,";//@period;;;;;;;;;;
+"all ""author"" elements with a ""period"" attribute. ,,";//author[@period = 'classical'] and //author[@period];;;;;;;;;;
+"the fourth ""author"" element. ,,";//author[4] and //author[.//name[text()='Alexander Pushkin']];;;;;;;;;;
+"the ""name"" node of the fourth author,,";//author[4]/name;;;;;;;;;;
+the name of the fourth author (text),,;//author[4]/name/text();;;;;;;;;;
+all elements that have any attributes,,;//*[@*];;;;;;;;;;
+all elements without any attributes,,;//*[not(@*)];;;;;;;;;;
+the node of author born in 1828. ,,;//author[@born='1828'];;;;;;;;;;
+name of the author born in 1828 (text),,;//author[contains(@born, '1828')]/name/text();;;;;;;;;;
+all authors that died after the year 0. ,,;//author[@died > 0];;;;;;;;;;
+the second author that died after the year 0 ,,;/authors/author[(number(attribute::died) >=0) and position()=2];;;;;;;//author[(@born < 0) and position() = 2];;;
+the name of second author that died after the year 0 (text),,;;;;;;;;//author[(@born < 0) and position() = 2]/name/text();;;
+"the last ""author"" element. ,,";/authors/author[last()];;;;;;;;;;
+"the ""name"" element that contains the text ""Tolstoy"". ,,";//*[text()='Leo Tolstoy'];;;;;;;//author/name[text()='Leo Tolstoy'];;;
```

Comment:
- 1st returns any element, need only 'name' element
 - please check contans, not equals condition